content/en/docs/tasks/security/authentication/auto-mtls/index.md

```diff
@@ -126,13 +126,14 @@ You can verify setup by sending an HTTP request with `curl` from any `sleep` pod
 For example, here is a command to check `sleep.full` to `httpbin.full` reachability:
 
 {{< text bash >}}
-$ kubectl exec $(kubectl get pod -l app=sleep -n full -o jsonpath={.items..metadata.name}) -c sleep -n full -- curl http://httpbin.full:8000/headers  -s  -w "response %{http_code}\n" | egrep 'Client-Cert.*sa.default|response.*$' -o
-Client-Cert": "By=spiffe://cluster.local/ns/full/sa/default
+$ kubectl exec $(kubectl get pod -l app=sleep -n full -o jsonpath={.items..metadata.name}) -c sleep -n full -- curl http://httpbin.full:8000/headers  -s  -w "response %{http_code}\n" | egrep -o 'URI\=spiffe.*sa/[a-z]*|response.*$'
```

Comment:
Or may be just grep "spiffe" to avoid other change in the future.

Reply:
The actual header is 
```
    "X-Forwarded-Client-Cert": "By=spiffe://cluster.local/ns/full/sa/httpbin;Hash=22c9623e6aac2e31f8720678d0816c8f43be3534b3c0094a0f6c21ce3ec90020;Subject=\"\";URI=spiffe://cluster.local/ns/full/sa/sleep"
```

Both client and server certs, kinda of verbose and confusing.

Reply:
Hmm, why it show the server SA. May be it's worth to check, outside the scope of this PR

Reply:
The second part is the client SA.